### PR TITLE
Update how-to-create-a-simple-binding.md

### PR DESCRIPTION
**Pull request by lymims**

### CODE REVIEW

**Comment by v-laluqu** [comment]
Hello @lymims 
Thank for your contribution. We are currently reviewing and will communicate the decision taken.
Thanks for your time and support on this.

Kind regards,
Microsoft C&E Localization Team

**Comment by ticianaciminari** [comment]
Hello, @lymims 

Thank you for your feedback and suggestions.
 
This proposed file change has been accepted. The suggestion will be merged to this repository.  We will release this article on the next localization release round. 
 
Many thanks for your support.

Kind regards,

Microsoft C&E Localization Team

File: docs/framework/wpf/data/how-to-specify-the-binding-source.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ ms.lasthandoff: 05/04/2018
   
 |属性|描述|  
 |--------------|-----------------|  
-|<xref:System.Windows.Data.Binding.Source%2A>|使用此属性将源设置为对象的实例。 如果你不需要建立一个作用域使其多个属性继承相同的数据上下文，则可以使用<xref:System.Windows.Data.Binding.Source%2A>属性而不是`DataContext`属性。 有关详细信息，请参阅<xref:System.Windows.Data.Binding.Source%2A>。|  
+|<xref:System.Windows.Data.Binding.Source%2A>|使用此属性将源设置为对象的实例。如果不需要特定功能（用于建立其中有多个属性继承相同的数据上下文的作用域），则可以使用 <xref:System.Windows.Data.Binding.Source%2A> 属性而不是 `DataContext` 属性。有关详细信息，请参阅 <xref:System.Windows.Data.Binding.Source%2A>。|
```
Comment (lymims) [inline docs/framework/wpf/data/how-to-specify-the-binding-source.md]: 请问 @ticianaciminari ，在<xref:Property>或`Property`的前后需要加空格吗？

Comment (imba-tjd) [reply docs/framework/wpf/data/how-to-specify-the-binding-source.md]: 中文和英文之间需要加空格，两者和标点之间不用。

Comment (lymims) [reply docs/framework/wpf/data/how-to-specify-the-binding-source.md]: 感谢

**Pull request merged**